herbert_core/admin/tmp_dir.m

```diff
@@ -6,19 +6,25 @@
 % tmp_dir     tempdir value on any machine  and (userpath()/tmp)
 %            (usually /home/user_name/Documents/MATLAB/tmp) folder if the machine is
 %            identified as iDaaaaS machine.
-%
-% $Revision:: 840 ($Date:: 2020-02-10 16:05:56 +0000 (Mon, 10 Feb 2020) $)
-%
 
 if is_idaaas()
```

Comment:
this `if` makes the code ISIS specific. Is there any way to use a standard approach across ALL platforms so we don't need this site-specific code branch?

Reply:
can not think of one. The iDaaaS issue is that two different machines (large and small) with different Horace configurations work from one users folder. There are also other strange issues we need to deal with, like this tmp directory, which can not be normal tmp directory as on normal OS. this is why is_idaaas was introduced and needs to be maintained. We also have idaaas specific configurations.